packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts

```diff
@@ -41,12 +45,20 @@ export const tooltipPositionVariants = styleVariants({
         left: '50%',
         transform: 'translate(-50%, -100%)',
       },
+      '::after': {
+        borderBottom: `6px solid ${token('color.background.base.inverse.default')}`,
+        borderLeft: '6px solid transparent',
+        borderRight: '6px solid transparent',
+        transform: 'translateX(50%)',
+        top: '-6px',
+        right: '50%',
+      },
```

Comment:
if you look closely, there is actually already an indicator, it just doesn't have the right color. I would recommend updating the color of that one rather than adding an additional pseudo element 
<img width="555" alt="image" src="https://github.com/kadena-community/kadena.js/assets/9022549/e9db5b6d-753a-4c12-b7a2-20565ed00c0c">